### PR TITLE
Updated Helm URLs and added devcontainer definition

### PR DESCRIPTION
**Pull request by harmen-xb**
I was trying out the Theia Cloud Try-Now Terraform and encountered some issues with Helm and also didn't want to install all dependencies locally on my machine. So I propose the following changes:
- Updated Helm URLs to eclipse-theia.github.io.
- Created devcontainer definition with all dependencies for the try-now deployment.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		// },
+		"ghcr.io/devcontainers/features/terraform:1": {},
+		// "ghcr.io/bartventer/arch-devcontainer-features/gcloud-cli:1": {}
+	},
+	
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "bash scripts/install-dependencies.sh"
```
Comment (jfaltermeier) [inline .devcontainer/devcontainer.json:26]: I had to adjust the command like this on my side:
```suggestion
	"postCreateCommand": "bash .devcontainer/scripts/install-dependencies.sh"
```

File: .devcontainer/scripts/install-dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+sudo apt-get update
+# Install dependencies for Google Cloud Client
+sudo apt-get install apt-transport-https ca-certificates gnupg curl
+# Install Google Cloud Client
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+sudo apt-get update && sudo apt-get install google-cloud-cli
+# Install Google Cloud Auth Plugin
+gcloud components install gke-gcloud-auth-plugin
```
Comment (jfaltermeier) [inline .devcontainer/scripts/install-dependencies.sh:9]: I had to install the GKE gcloud auth plugin as well:
`sudo apt-get install google-cloud-cli-gke-gcloud-auth-plugin`

**Review by jfaltermeier — COMMENTED** [review]
Thank you very much! The helm repository updates are working as expected. Regarding the dev container, I had to make two adjustments on my side, which I've detailed in inline comments. Could you please review them?

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (harmen-xb) [reply .devcontainer/devcontainer.json:26]: That makes sense indeed, I wrote the scripts while running into dependency issues. I thought I rebuild the container afterwards, but apparently not.

File: .devcontainer/scripts/install-dependencies.sh (hunk 2 above)
Comment (harmen-xb) [reply .devcontainer/scripts/install-dependencies.sh:9]: You are right, I have this installed as well, forgot to add it in the bash scripts.

File: .devcontainer/scripts/install-dependencies.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+sudo apt-get update
+# Install dependencies for Google Cloud Client
+sudo apt-get install apt-transport-https ca-certificates gnupg curl
+# Install Google Cloud Client
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo gpg --dearmor -o /usr/share/keyrings/cloud.google.gpg
+echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+sudo apt-get update && sudo apt-get install google-cloud-cli
```
Comment (jfaltermeier) [inline .devcontainer/scripts/install-dependencies.sh:7]:
```suggestion
sudo apt-get update && sudo apt-get install google-cloud-cli google-cloud-cli-gke-gcloud-auth-plugin
```

**Review by jfaltermeier — APPROVED** [review]

**Pull request merged**